08-walkthrough-3.Rmd

```diff
@@ -211,83 +256,60 @@ rf_fit2 <- train(final_grade ~ .,
 rf_fit2
 ```
 
-The model with the same values as identified before but with **min.node.size** equal to 1 seems to fit best, though the improvement seems to be fairly small relative to the difference the other tuning parameters seem to make. 
+The model with the same values as identified before for **mtry** and **splitrule**, but with **min.node.size** equal to 1 seems to fit best, though the improvement seems to be fairly small relative to the difference the other tuning parameters seem to make. 
 
```

Comment:
it would be helpful to explain how you know that this model fits best

Reply:
Definitely - thanks